### ISSUE TITLE
Feature request: allow more flexibility for key value type

### ISSUE THREAD

**Issue by nathanfoon** [jrmykolyn/sfco-path-map#1]
The docs clearly state that the value of a key must be a string; but it would be great if the value could be something other than a string.

For example, I'm in a situation that requires a key's value to be an array of strings (see `scriptsSrc` key):
```js
const PATHS = new PathMap( {
  projectRoot: 'path/to/project',
  scriptsSrc: [
    '{{projectRoot}}/js/src/*.js',
    '!{{projectRoot}}/js/src/ignore-me.js'
  ],
  scriptsDest: '{{projectRoot}}/js/src/',
} );
```

**Comment by jrmykolyn** [comment]
@nathanfoon 

Nice flag, and good idea. Will update `PathMap` definition to support/handle both `string` and `array` values.

**Comment by jrmykolyn** [comment]
@nathanfoon 

Feature has been implemented in `0.4.0` release. Please update your local package, and let me know if you notice any issues.

Updates to documentation is_remain.
